### PR TITLE
WL-364 : adding unit tests

### PR DESCRIPTION
**Pull request by massa84**

### CODE REVIEW

File: test/spec/api/documents.spec.js [hunk 1]
```diff
@@ -299,6 +305,47 @@ define(function(require) {
 
         });
 
+        describe('clone method', function() {
+
+            var ajaxRequest,
+                apiRequest;;
```
Comment (customcommander) [inline test/spec/api/documents.spec.js]: double `;;`

File: test/spec/api/documents.spec.js [hunk 2]
```diff
@@ -299,6 +305,47 @@ define(function(require) {
...
+            it('should use endpoint /documents/{id}/actions/cloneTo', function() {
+                expect(ajaxRequest.url).toBe(baseUrl + '/documents/15/actions/cloneTo');
+            });
+
+            it('should have a Content-Type header', function() {
+                expect(ajaxRequest.headers['Content-Type']).toBeDefined();
+            });
+
+            it('should have an Authorization header', function() {
+                expect(ajaxRequest.headers.Authorization).toBeDefined();
```
Comment (customcommander) [inline test/spec/api/documents.spec.js]: Do you need this one if you then check for a specific value?

File: test/spec/api/documents.spec.js (hunk 1 above)
Comment (massa84) [inline test/spec/api/documents.spec.js]: fixed

File: test/spec/api/documents.spec.js (hunk 2 above)
Comment (massa84) [inline test/spec/api/documents.spec.js]: Copy pasted from other tests as it looks like the standard test for the auth header. 
If we want we can change it everywhere.

Comment (customcommander) [inline test/spec/api/documents.spec.js]: Nope that's fine.

**Pull request merged**